packages/wonder-blocks-core/components/view.js

```diff
@@ -42,7 +42,11 @@ const StyledDiv = addStyle<"div">("div", styles.default);
  */
 export default class View extends React.Component<TextViewSharedProps> {
     render() {
-        const {testId, ...restProps} = this.props;
+        if (this.props.tag) {
+            StyledDiv = addStyle<string>(this.props.tag, styles.default);
```

Comment:
Instead of overriding the variable, I think we should use a different one and do something like.

```
const StyledTag = tag ? addStyle... : StyledDiv;
```
(`addStyle...` being a placeholder in this example for whatever the call is with the appropriate type for allowed tags)
 
Then render with `StyledTag`.

What do you think?